build/ci/library_owners_jira.json

```diff
@@ -1,4 +1,4 @@
 {
-	"mongocli": "22223",
+	"mongocli": "27065",
```

Comment:
updated team ID to be the new team

Reply:
keeping mongocli since the team still doesn't have a team ID

Reply:
should we split apix-2 and mongocli?

Reply:
we can't simply say they own all dependencies of the repo, right?

Reply:
do you mean handover the mongocli team to them? I'm not sure how much logic we have depending on that team so not sure

Reply:
I think we will need another PR in `master` to replace `mongocli` with `apix-2`

Reply:
I think what Filipe means is not using this file, we created this logic in master because there were k8s libs we didn't own but on this branch everything is owned by EA

Reply:
ohh ok, so for mongocli we can just put EA

Reply:
yes

Reply:
what @gssbzn said plus a review in https://github.com/mongodb/mongodb-atlas-cli/pull/3206